core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/transfer/ProvisionCallbackDelegate.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (c) 2020 - 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.transfer.core.transfer;
+
+import org.eclipse.dataspaceconnector.spi.transfer.provision.DeprovisionResult;
+import org.eclipse.dataspaceconnector.spi.transfer.provision.ProvisionResult;
+
+import java.util.List;
+
+public interface ProvisionCallbackDelegate {
```

Comment:
Do we need this interface? I would keep this methods on the `TransferProcessManager` interface until a specific need will come to the surface.

Reply:
Yes, for the HTTP Provisioner (coming soon)